config.py

```diff
@@ -528,7 +528,7 @@ def wait_for_previous(self):
                 f'#SBATCH --account={self.compute_account}',
                 f'#SBATCH --partition={self.compute_queue}',
                 f'#SBATCH --constraint={self.constraint}',
-                f'#SBATCH --dependency=afterok:{dep_str}', '', '# Do nothing',
+                f'#SBATCH --dependency=afterany:{dep_str}', '', '# Do nothing',
```

Comment:
@leclairm I don't understand this change. I have the feeling this breaks some of the functionality.